--- a/src/cli/cli.c
+++ b/src/cli/cli.c
@@ -131,7 +131,7 @@
 	/* termux-exec replaced execve with path with one that doesn't exist inside proot?  */
 	if (status == -ENOENT && getenv("LD_PRELOAD") != NULL && strstr(getenv("LD_PRELOAD"), "libtermux-exec.so") != NULL) {
 		note(tracee, INFO, USER,
-"It seems that termux-exec is active and is prepending /data/data/com.termux/... to executable paths\n"
+"It seems that termux-exec is active and is prepending @TERMUX_PREFIX@/... to executable paths\n"
 "If this is path is not available inside proot, please \"unset LD_PRELOAD\"");
 		return;
 	}
